README.md

```diff
@@ -134,16 +134,20 @@ In addition to containerd, the following components should be installed:
 
 These dependencies are included in `nerdctl-full-<VERSION>-<OS>-<ARCH>.tar.gz`, but not included in `nerdctl-<VERSION>-<OS>-<ARCH>.tar.gz`.
 
-### Brew
```

Comment:
Why remove?

Reply:
I just renamed to `Linux` the other items are based on the OS types instead, i.e.  Windows, FreeBSD, MacOS.

Reply:
@jsturtevant -- I've replaced that with the standard installation process on Linux. I don't see a reason to install `brew` first so that you can install `nerdctl` when most likely they have already installed `containerd`, etc. using the standard process of getting the binaries from Github with `curl`.